src/webdav4/urls.py

```diff
@@ -28,7 +28,7 @@ def join_url(
 
 def join_url_path(hostname: str, path: str) -> str:
     """Returns path absolute."""
-    path = path.strip("/")
+    path = path.lstrip("/")
```

Comment:
Let's keep this unchanged for now.
```suggestion
    path = path.strip("/")
```